lib/transaction.js

```diff
@@ -26,6 +30,19 @@ var Transaction = module.exports = function(sequelize, options) {
   } else {
     this.id = this.name = Utils.generateUUID();
   }
+
+  // `transaction.promise` implementation #2805
+  // main propose is to allow chainable logic inside hooks
```

Comment:
I love Portuguese, but i wish that English is a kind of second native language :p